sympy/vector/coordsysrect.py

```diff
@@ -14,7 +15,7 @@ class CoordSysCartesian(Basic):
     """
 
     def __new__(cls, name, location=None, rotation_matrix=None,
-                parent=None, vector_names=None, variable_names=None):
+                parent=None, vector_names=None, variable_names=None, curv_coord_name=None, transformation_equations=None):
```

Comment:
I would only have one additional parameter here. What about an element of the class you are adding?